### DISCUSSION TITLE
Pre-installed extension missing

### DISCUSSION THREAD

**Discussion by quentingosset** [shinsenter/php#171] [Q&A] [answered]
Hello, after upgrading to latest php-fpm containter i see the pre-installed extension is missing (on my view is already see gd;zip;pdo_mysql).
A exemple, php-gd is missing : 
![image](https://github.com/user-attachments/assets/417789ea-8e28-4a5c-8c9c-3a008bf9ac8e)

**Comment by shinsenter** [comment, marked as answer]
Hello @quentingosset ,

PHP 8.4.1 has now been officially released, so the `latest` tag for the Docker images will also switch to PHP 8.4. However, some extensions are still not fully compatible with this version. You can use the tags for PHP 8.3 instead of the `latest` tag if needed.
